src/components/About.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useRef } from 'react'
+import './About.css'
+
+const AboutModal = ({showModal, setShowModal}) => {
+    // Closes modal when user clicks outside of video box
+    const modalRef = useRef()
+    const closeModal = e => {
+        if (modalRef.current === e.target) {
+            setShowModal(false)
+        }
+    }
+
+    return (
+        <>
+            {showModal ? (
+                <div className="modal-wrapper" ref={modalRef} onClick={closeModal}>
+                    <div className={"about-modal"}>
+                        <div className="modal-header">
+                            <i className="fas fa-times modal-close" onClick={() => setShowModal(false)}></i>
+                        </div>
+                        <div className="modal-content">
+                            <iframe className="about-vid" title="SBUHacks 2018" frameborder="0" allowfullscreen="" src="https://www.youtube.com/embed/6vjZo2qCSrw?rel=0&amp;showinfo=0&amp;autoplay=1"></iframe>
+                        </div>
+                    </div>
+                </div>
+            ) : null}
+        </>
+    )
+}
+
+function About() {
```

Comment:
You switch between using arrow functions and normal functions everywhere. Stick to one everywhere. Change this in all your files.